contribs/gmf/less/layertree.less

```diff
@@ -30,22 +45,35 @@ gmf-layertree ul .treenode {
     }
   }
 
-  .rightButtons *::after {
-    background: none;
-    color: @map-tools-color;
-    display: inline;
-    font-family: FontAwesome;
+  .rightButtons {
+    float: right;
+
+    *::before {
+      background: none;
+      position: relative;
+      right: initial;
+    }
+
+    *::after {
+      background: none;
+      position: relative;
+      right: initial;
```

Comment:
`background`, `position`, `right` is the same for `before` and `after`. Why not factorizing?